www/base/guanlecoja/config.coffee

```diff
@@ -3,7 +3,7 @@
 #   This module contains all configuration for the build process
 #
 ### ###############################################################################################
-ANGULAR_TAG = "~1.3.1"
+ANGULAR_TAG = "~1.4.4"
```

Comment:
It's not yet merged, something was problematic with it: https://github.com/buildbot/buildbot/pull/1799